datafusion/core/tests/sqllogictests/test_files/pg_compat_simple.slt

```diff
@@ -15,6 +15,67 @@
 # specific language governing permissions and limitations
 # under the License.
 
+###
+## Setup test for postgres
+###
+
+onlyif postgres
```

Comment:
I used the `onlyif` statement to run different setups for DataFusion and posgres